Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI_Tests.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+@available(iOS 13.0, *)
+class ChatChannelListVCComponent_Tests: iOS13TestCase {
+    var chatChannelList: ChatChannelList!
+    var mockedChannelListController: ChatChannelListController_Mock<NoExtraData>!
+
+    override func setUp() {
+        super.setUp()
+        mockedChannelListController = ChatChannelListController_Mock.mock()
+        chatChannelList = ChatChannelListVC.View(controller: mockedChannelListController)
+    }
+
+    func test_chatChannelList_isPopulated() {
+        let channels = chatChannelsMock()
+
+        mockedChannelListController.simulateInitial(
+            channels: channels,
+            state: .localDataFetched
+        )
+
+        AssertSnapshot(chatChannelList)
```

Comment:
Should we test these inside a navigation view? Like we do it with the VC?

Reply:
I have added `isEmbeddedInNavigationController: true`, will that suffice for now? The thing is that navigation controller will be there, but it will be empty.